index.html

```diff
@@ -6,11 +6,50 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <header>
+        <img class="logo" src="/img/Logo.png" alt="Baket sweet logo">
+        <p>The best cakes in town delivered to your door</p>
+        <nav>
+            <ul>
+                <li><a href="">Home</a></li>
```

Comment:
I know this homework is mainly about responsive design but the whole point of having the nav bar is easier navigation to relevant parts of the page, you should consider how make it so clicking the a tag moves you to the right part of the page